pkg/driver/driver.go

```diff
@@ -16,6 +16,14 @@ type Driver interface {
 	// Validate returns error if the current driver isn't support for given config
 	Validate() error
 
+	// Initialize is called on creating the instance for initialization.
+	// (e.g., creating "vz-identifier" file)
+	//
+	// Initialize must not return a non-nil error when it is called against an existing instance.
```

Comment:
Not a fan of double-negation

```suggestion
	// Initialize MUST return nil when it is called against an existing instance.
```

Reply:
done